python/ray/remote_function.py

```diff
@@ -187,8 +188,16 @@ def _remote(self,
             raise ValueError("Non-direct call tasks are no longer supported.")
         if max_retries is None:
             max_retries = self._max_retries
-        if placement_group_id is None:
-            placement_group_id = ray.PlacementGroupID.nil()
+
+        if placement_group is None:
+            placement_group = PlacementGroup(ray.PlacementGroupID.nil(), -1)
+            if placement_group_bundle_index != -1:
+                raise ValueError("If placement group is not set, "
+                                 "the value of bundle index must be -1.")
+        elif placement_group_bundle_index >= placement_group.bundle_count \
+                or placement_group_bundle_index < -1:
+            raise ValueError("placement group bundle index {} is invalid."
```

Comment:
Can you write a better message? Also, please use f string! (We are moving away from .format since we dropped Python 3.5 support). 

`f`placement group bundle index {placement_group_bundle_index} is invalid. Valid placement group indexes: 0-{placement_group.bundle_count}`

Reply:
fixed